core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -56,6 +62,21 @@ var ExplorationEditorHistoryTab = function() {
    * Workflows
    */
 
+  this.expectCommitDatesToBeDisplayed = async function() {
+    /*
+     * This method checks if the commit dates are being displayed in
+     * List of Changes section of the history tab.
+     */
+    var numCommitDates = await datesCommitsWereSaved.count();
+    for (var i = 0; i < numCommitDates; i++) {
+      var date = await (await datesCommitsWereSaved.get(i)).getText();
+      // The dates can be of varying format. To play it safe and to
+      // keep it simple, we will just check if the date string contains
+      // a digit.
```

Comment:
What kinds of formats are possible? For this test, the first commit will have been created seconds ago at the longest, so does that narrow down the possible date formats?

Reply:
It's based off of getLocaleAbbreviatedDatetimeString in date-time-format.service.ts. Some examples are MM/DD/YY (02/02/20), MMM DD (Feb 2) and HH:MM AM/PM (7:01 AM). I considered just testing the hour and minute format like you're suggesting but thought it would be better to play it safe and keep it simple, especially since multiple formats would be needed if we were testing multiple commit dates. @seanlip was mentioning there could be some issues relating to time zones?

Reply:
Yeah, I think keeping it simple is probably best, since we don't want this test to fail in different locales. I also started by thinking that we could assume it would be seconds, but unfortunately it's possible for the commit to have been created just before midnight on the previous day, and to be checked just after midnight on the current day -- so probably best to err on the side of generality IMO. What do you think @U8NWXD?

Reply:
I think this is reasonable!